test/routes/index-test.js

```diff
@@ -12,10 +12,15 @@ describe('Index page', () => {
         await server.stop();
     });
 
-    it('should show "Hello Node.js" message', async () => {
+    it('should return a 200 response', async () => {
```

Comment:
It's all OK, but I'm just curious why did you split the test in two? For me, it was quite logical to assert the response for its content and its status code in the same place.

Reply:
I thought that it apply rule that a unit test should test only one thing.